src/clients/SpokePoolClient.ts

```diff
@@ -404,17 +420,16 @@ export class SpokePoolClient extends BaseAbstractClient {
     }
 
     // Order fills by totalFilledAmount and then return the first fill's full deposit amount minus total filled amount.
-    const fillsOrderedByTotalFilledAmount = validFills.sort((fillA, fillB) =>
-      fillB.totalFilledAmount.gt(fillA.totalFilledAmount)
-        ? 1
-        : fillB.totalFilledAmount.lt(fillA.totalFilledAmount)
-        ? -1
-        : 0
-    );
+    const fillsOrderedByTotalFilledAmount = validFills.sort((fillA, fillB) => {
+      const totalFilledA = getTotalFilledAmount(fillA);
+      const totalFilledB = getTotalFilledAmount(fillB);
+
+      return totalFilledB.gt(totalFilledA) ? 1 : totalFilledB.lt(totalFilledA) ? -1 : 0;
+    });
```

Comment:
This change is added as a simple way to silence tsc warnings. In practice we should never see v3 fills here because they guarantee to fill to completion, but tsc doesn't know that.